src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -790,6 +790,76 @@ static char[] Resize(char[] buffer)
             }
         }
 
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
```

Comment:
Would it add value to add a `text` column to the test as well?

Reply:
Probably not, that is handled by TryReadStringWithEncoding which is already continue-aware.